index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mother Earth's Plantasia</title>
+</head>
+
+<body>
+    <div id="menu-background" class="">
+        <button id="quit">
+        </button>
+        <ul id="menu-items" class="">
+            <li>
+                <a href="index.html"><img src="images/Plantasia-logo-inner-background.png" alt="Plantasia logo word"></a>
+            </li>
+            <li>
+                <a href="#intro">ABOUT</a>
+            </li>
+            <li>
```

Comment:
- [ ] For now we can access `About` Page using `More of Placantia` link but it would be better to link this page also to `About` tab.

Reply:
- [ ] As requested in the project requirements, please deploy the application using `github pages` and the link to your readme file.

Reply:
Hello @uwadonat, regarding the first comment, can you rephrase it? I do not understand what you mean. You mean, change the href of the <a> tag with the value ABOUT to direct me to the about.html file (That is what About tab meant?)?
Note that the about <a> directs the user to the intro section, also, will it be ok having two links? What do I do with More Of plantasia link? Do I leave it blank?

Reply:
Done! Changed the `<a href="#intro">ABOUT</a>` to `<a href="html/about.html">ABOUT</a>`

Reply:
Updated the README.md file to have the live demo link on the corresponding section.